src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/model/in/CaseUpdate.java

```diff
@@ -5,14 +5,29 @@
 
 public class CaseUpdate {
 
+    public final boolean isAutomatedProcess;
     public final ExceptionRecord exceptionRecord;
+    public final CaseUpdateDetails caseUpdateDetails;
     public final CaseDetails caseDetails;
 
+    /**
+     * Used for 2 case update endpoints:
+     * <ul>
+     *     <li>Exception record case update</li>
+     *     <li>Auto case update</li>
+     * </ul>
+     * <i>case_update_details</i> element is added for Auto Case update request.
+     * <i>exception_record</i> element can be removed after moving to the Auto case update endpoint.
```

Comment:
This is actually one endpoint.

Maybe it would be best to just have a comment next to `exeptionRecord` field, saying it will go away? `case_update_details` is the proper one to use, regardless of the scenario and `exception_record` is only kept for now in order to prevent breaking changes and offer smooth transition to services.

Reply:
ok